packages/perseus/src/perseus-types.ts

```diff
@@ -1059,17 +1060,6 @@ export type PerseusLabelImageWidgetOptions = {
     static: boolean;
 };
 
-export type PerseusLabelImageMarker = {
```

Comment:
Further, because TypeScript types match if they are the same shape, I was thinking about ScoringData and ValidationData for each widget as a matching subset of the widget options in this file.

Reply:
Do you mean use the types in this file to define the ScoringData and ValidationData types?